app/plugins/eu.esdihumboldt.hale.app.transform/src/eu/esdihumboldt/hale/app/transform/ExecApplication.groovy

```diff
@@ -150,27 +150,36 @@ $baseCommand
      -stacktrace
      -trustGroovy
 
-  You can provide multiple sources for the transformation. If the source is a
-  directory, you can specify multiple -include and -exclude parameters to
-  control which files to load.
-  If you do not specify -include, it defaults to "**", i.e. all files being
-  included, even if they are in sub-directories.
-  Patterns use the glob pattern syntax as defined in Java and should be quoted
-  to not be interpreted by the shell, see
-  http://docs.oracle.com/javase/8/docs/api/java/nio/file/FileSystem.html#getPathMatcher-java.lang.String-
-
-  -filter, -filter-on and -exclude-type are different types filter, applied on source data.
-  If you specify more than one filter ( e.g. -filter or -filter-on) will be applied as 'OR' on each instance.   
-  If instance matches any of the filter, will be included for transformation.
-  If instance matches -exclude-type filter, then it will be excluded from the transformation, irrespective of trueness of other filters.
-
-  You can also specify all the parameters in file using -args-file parameter. File content should follow specific rule. e.g. 
-  -param1
-  value
-  -param2
-  value
-  ...
-  ...""".trim()
+  Sources
```

Comment:
So you included my changes from #86 in your pull request? Not really what was intended, but you can leave it in here.